wemake_python_styleguide/visitors/ast/keywords.py

```diff
@@ -122,3 +123,26 @@ def visit_Try(self, node: ast.Try) -> None:
         """Used for find finally in try blocks without except."""
         self._check_for_needs_except(node)
         self.generic_visit(node)
+
+
+class WrongExceptionTypeVisitor(BaseNodeVisitor):
+    """Finds use of ``BaseException`` exception."""
+
+    def _check_exception_type(self, node: ast.ExceptHandler) -> None:
+        exception_name = getattr(node, 'type', None)
+        if exception_name is None:
+            return
+        exception_id = getattr(exception_name, 'id', None)
+        if exception_id == 'BaseException':
```

Comment:
Can we have a class-level variable (typed as `ClassVar[str]`) that will contain this value (`'BaseException'`)?

I like moving everything into own variables.

Reply:
Since none of existing classes had `init` constructor, I've moved the variable into a class variable that can be accessed by `ClassName.Variable`. Hope this is alright.